src/Microsoft.TestPlatform.CrossPlatEngine/EventHandlers/NullPathConverter.cs

```diff
@@ -0,0 +1,67 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.VisualStudio.TestPlatform.CommunicationUtilities;
+using System.Collections.Generic;
+using ObjectModel;
+using Microsoft.VisualStudio.TestPlatform.ObjectModel;
+using Microsoft.VisualStudio.TestPlatform.ObjectModel.Client;
+using System.Collections.ObjectModel;
+
+internal class NullPathConverter : IPathConverter
```

Comment:
```suggestion
internal class NopPathConverter : IPathConverter
```

Reply:
We have a mix of NoOp and Null in the code base. Keeping null.